sources/packages/backend/package.json

```diff
@@ -85,13 +85,11 @@
     "@nestjs/cli": "^9.3.0",
     "@nestjs/schematics": "^9.0.3",
     "@nestjs/testing": "^9.1.2",
-    "@types/bull": "^4.10.0",
```

Comment:
just for info, was this and line 94 deleted as part of any package update

Reply:
As the ticket description, there was a warning for that. This package `@types/bull` is not needed anymore as the own warning mentioned.